default/stringtables/en.txt

```diff
@@ -9553,22 +9553,22 @@ BLD_TERRAFORM
 Terraforming
 
 BLD_TERRAFORM_DESC
-Terraforms a world one step closer to the environmental preference of the inhabiting species.
+Terraforms a world one step closer to the [[encyclopedia ENVIRONMENT_TITLE]]al preference of the inhabiting species.
 
 BLD_TERRAFORM_REVERT
 Terraforming Reversion
 
 BLD_TERRAFORM_REVERT_DESC
-Terraforms a world one step back towards its original type.
+Terraforms a world one step back towards its original type of [[encyclopedia ENVIRONMENT_TITLE]].
 
 BLD_REMOTE_TERRAFORM
 Remote Terraforming
 
 BLD_REMOTE_TERRAFORM_DESC
-Terraforms a world one step closer to the environmental preference of the inhabiting species. This process can be enacted at an [[encyclopedia OUTPOSTS_TITLE]].
+Terraforms a world one step closer to the [[encyclopedia ENVIRONMENT_TITLE]]al preference of the inhabiting species. This process can be enacted at an [[encyclopedia OUTPOSTS_TITLE]].
```

Comment:
Errrr... that's weird. I forgot to remove an -al- on line 9568, so I updated again the PR with the same process, but this time, it marks the previous comments as based on an outdated diff...

I don't undertstand (I obviously refreshed the page the first time...)...